### ISSUE TITLE
Scapular compression test positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11041]
**Preferred term label:**
Scapular compression test positive

**Synonyms**
Scapular Assistance Test

**Definition (free text, please give PubMed ID)**
Examiner uses one hand to apply a posterior to anterior force is directly over the scapula and the anterior chest wall is stabilized with the other hand. Test is positive if this scapular stabilization allows for increased shoulder flexion compared to when no stabilization is applied. Also known as scapular assistance test.

https://www.physio-pedia.com/Scapular_Assistance_Test

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scapular winging

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as HP:0020334
Positive scapular compression test
